src/CommitmentsV2/SFA.DAS.CommitmentsV2.Messages/Events/IApprenticeshipDeleted.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.CommitmentsV2.Messages.Events
+{
+    public interface IApprenticeshipDeletedEvent
+    {
+        long Apprenticeship { get; set; }
+        long CommitmentId { get; set; }
+        string Uln { get; set; }
+        Guid? ReservationId { get; set; }
+        DateTime? CourseStartDate { get; set; }
+        string CourseCode { get; set; }
+        DateTime Deleted { get; set; }
```

Comment:
This variable isn't overly clearly named, can it be renamed to something like DeletedDateTime?